build_ngtf.py

```diff
@@ -144,8 +146,8 @@ def main():
 
     assert not (
         arguments.use_tensorflow_from_location != '' and
-        arguments.use_prebuilt_tensorflow != ''
-    ), "\"use_tensorflow_from_location\" and \"use_prebuilt_tensorflow\" "
+        (arguments.use_prebuilt_tensorflow != '')
```

Comment:
Is this change needed?

Reply:
I think it's accidental change and not needed.